### ISSUE TITLE
Copying non-ASCII characters from Windows to WSLg broken

### ISSUE THREAD

**Issue by squeaky-pl** [microsoft/wslg#20]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.21364.1]
Your Distribution version: Ubuntu 20.04
Your WSLg version: 1.0.17.1
```

# Steps to reproduce

Try opening notepad.exe and type non-ASCII string. Copy it and try pasting it inside gedit or gvim.

https://user-images.githubusercontent.com/754356/115618059-75f97480-a2f2-11eb-8b03-093a0cdff257.mp4

**WSL logs**: 

* [weston.log](https://github.com/microsoft/wslg/files/6353798/weston.log)
* `versions.txt`

```
WSLg ( x86_64 ): 1.0.17+3.Branch.master.Sha.a526dfd5ad03d126bb2d8c528f6c3563e86a40da
Mariner: VERSION="1.0.20210224"
FreeRDP: e4a2fc2053bd8c5f99455fcd08ffee7e5591567a
weston: fd961f5cd116c9358d82ce94d139c1578e21bd00
pulseaudio: 2f0f0b8c3872780f15e275fc12899f4564f01bd5
mesa:
````

#  Expected behavior

Copying non-ASCII characters between Windows and WSLg should work

# Actual behavior

In this example I am copying a string "ááá" from notepad.exe to gedit and gvim. In both cases gedit and gvim seem to expect UTF-8 encoded data but it looks like they receive ISO-8859-1 (latin1) where "á" is encoded as 0xE1.

**Comment by hideyukn88** [comment]
Yes, we are aware of this, the possible solution is to support conversion between CF_UNICODETEXT (at Windows) and utf-8 (and optionally utf-16) in WSLg side.

**Comment by hideyukn88** [comment]
Fixed in 1.0.23.
